### PR TITLE
ECC-12 - stop using app config for service name in tax enrolment

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/uk/gov/hmrc/customs/managesubscription/services/SubscriptionCompleteBusinessService.scala [hunk 1]
```diff
@@ -79,7 +79,7 @@ class SubscriptionCompleteBusinessService @Inject() (
     } yield (): Unit
 
   private def dataStoreEmailRequest(recipient: RecipientDetailsWithEori)(implicit hc: HeaderCarrier): Future[Unit] =
-    if (recipient.recipientDetails.service == "cds")
+    if (recipient.recipientDetails.service == "HMRC_CUS_ORG")
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/customs/managesubscription/services/SubscriptionCompleteBusinessService.scala:82]: This is the only place where "recipientDetails.service" is actually being used.  So this switch in logic is quite straight forward.

File: test/unit/services/EmailServiceSpec.scala [hunk 2]
```diff
@@ -34,6 +34,8 @@ class EmailServiceSpec extends BaseSpec {
 
   private val mockEmailConnector = mock[EmailConnector]
 
+  private val service = "atar"
```
Comment (SteveSugden) [inline test/unit/services/EmailServiceSpec.scala:37]: I'll re-use this val

File: app/uk/gov/hmrc/customs/managesubscription/services/SubscriptionCompleteBusinessService.scala (hunk 1 above)
Comment (PatrykRudnicki) [reply app/uk/gov/hmrc/customs/managesubscription/services/SubscriptionCompleteBusinessService.scala:82]: It should be `HMRC-CUS-ORG`.

I'm wondering if we should name this val in the recipientDetails from service to enrolment key

File: test/unit/services/EmailServiceSpec.scala [hunk 3]
```diff
@@ -34,6 +34,8 @@ class EmailServiceSpec extends BaseSpec {
 
   private val mockEmailConnector = mock[EmailConnector]
 
+  private val service = "HMRC-ATAR_ORG"
```
Comment (PatrykRudnicki) [inline test/unit/services/EmailServiceSpec.scala:37]: and Here it should be `HMRC-ATAR-ORG` not `HMRC-ATAR_ORG`

File: test/unit/services/SubscriptionCompleteBusinessServiceSpec.scala [hunk 4]
```diff
@@ -65,7 +65,7 @@ class SubscriptionCompleteBusinessServiceSpec
 
   private val cdsRecipientDetails: RecipientDetails = RecipientDetails(
     Journey.Subscribe,
-    "cds",
+    "HMRC_CUS_ORG",
```
Comment (PatrykRudnicki) [inline test/unit/services/SubscriptionCompleteBusinessServiceSpec.scala:68]: In those two cases the same `-` instead of `_`

**Review by PatrykRudnicki — COMMENTED** [review]
Additionally can you remove the config from application.conf

**Review by PatrykRudnicki — APPROVED** [review]

**Pull request merged**